integration-tests/script/docker_compose_args.sh

```diff
@@ -57,6 +61,10 @@ getComposeArgs()
     then
       # the 'high availability' test cluster with multiple coordinators and overlords
       echo "-f ${DOCKERDIR}/docker-compose.high-availability.yml"
+    elif [ "$DRUID_INTEGRATION_TEST_GROUP" = "kafka-data-format" ]
+    then
```

Comment:
Maybe add a comment above so it doesn't look like this was an accidental empty branch

Reply:
this is just matching the style of the other comment blocks in this script so going to leave like it is to be consistent